core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTestBase.scala

```diff
@@ -150,7 +150,7 @@ trait SplitClusterDUnitTestBase extends Logging {
     doTestRowTableCreation()
   }
 
-  def _testComplexTypesForColumnTables_SNAP643(): Unit = {
+  def testComplexTypesForColumnTables_SNAP643(): Unit = {
```

Comment:
Please override it as no-op in SplitClusterDUnitSecurityTest, else test suite may fail. See SplitClusterDUnitSecurityTest.testTableFormChanges()